workspaces/capabilities.go

```diff
@@ -77,6 +77,14 @@ func (g CapabilitiesGenerator) Generate(runConfig types.RunConfig) error {
 
 func (g CapabilitiesGenerator) transformCapabilities(runConfig types.RunConfig) (types.CapabilityConfigs, error) {
 	capabilities := runConfig.Capabilities
+	if len(runConfig.NamedCapabilities) > 0 {
```

Comment:
I guess if NamedCapabilities are empty (because there are none) it won't matter. Just pointing out that you can't tell the difference between uninitialized and intentionally empty.